containers/python-examples/NLTK/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3.4"
+
+services:
+  server:
+    image: ghcr.io/deephaven/server-nltk:${VERSION:-latest}
+    expose:
+      - '8080'
+    volumes:
+      - ./data:/data
+      - api-cache:/cache
+    environment:
+      - JAVA_TOOL_OPTIONS=-Xmx4g -Ddeephaven.console.type=python
+
+  web:
+    image: ghcr.io/deephaven/web:${VERSION:-latest}
+    expose:
+      - '80'
+    volumes:
+      - ./data:/data
+      - web-tmp:/tmp
+
+  grpc-proxy:
+    image: ghcr.io/deephaven/grpc-proxy:${VERSION:-latest}
+    environment:
+      - BACKEND_ADDR=grpc-api:8080
```

Comment:
This won't work, we don't have `grpc-api` anymore. This makes me think that these compose files haven't been tested? All docker container references to `grpc-api` should be changed as appropriate to `server` where applicable. Same goes for other compose files.